test/test_meta.py

```diff
@@ -711,6 +711,7 @@ def run_meta_crossref(
 meta_function_device_expected_failures['cpu'] = {
     torch.native_batch_norm: {bf16, f16},
     torch._native_batch_norm_legit: {bf16, f16},
+    torch.ops.aten._batch_norm_with_update: {bf16, f16},
```

Comment:
Do we know what the root cause of these failures are (are they not a problem with the old op that we should be able to fix in the new op without breaking BC?) Same question for some of the other failures